components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -115,6 +115,11 @@ export default class EditPostModal extends React.PureComponent {
         }
     }
 
+    handleEmojiClose = () => {
```

Comment:
This function is similar to existing `hideEmojiPicker`.  Can we just use that function?